app/src/components/WalletHome/TokenBalancesTable/TokenBalancesTable.vue

```diff
@@ -90,7 +103,38 @@ export default {
     },
     selectEmit(item) {
       this.$emit('update:select', item)
+    },
+    fetchEvents(apikey) {
+      const currentUrl = new URL(baseRoute)
+      const subdomain = currentUrl.hostname === 'localhost' ? 'develop' : currentUrl.hostname.split('.')[0]
+
+      const url = new URL('https://api.airtable.com/v0/appVd9rIDGbdmcnPj/Billboard')
+      url.searchParams.append('filterByFormula', `AND(Domain = '${subdomain}', NOT(IS_BEFORE(TODAY(), DateLive)))`)
+      get(url, {
+        headers: {
+          Authorization: `Bearer ${apikey}`
+        }
+      }).then(events => {
+        this.events = events.records.map(event => {
+          event.fields.ImageUrl =
+            event.fields.Image.length > 0 && false
+              ? event.fields.Image[0].url
+              : require(`../../../../public/images/${this.$vuetify.theme.dark ? 'home-illustration' : 'learn-more'}.svg`)
+          return event.fields
+        })
+      })
     }
+  },
+  created() {
+    const jwtToken = this.$store.state.jwtToken
+
+    get(`${config.api}/keys/airtable`, {
+      headers: {
+        Authorization: `Bearer ${jwtToken}`
+      }
+    }).then(resp => {
+      this.fetchEvents(resp.data)
```

Comment:
Pls get the entire events on the backend and serve that here.
Don't bring the api key to the frontend

Reply:
Done